README.md

```diff
@@ -12,7 +13,10 @@ Go client for Pilosa high performance distributed bitmap index.
 
 ## Change Log
 
-* **Next**
+* **v0.8.1** (2017-02-28)
+    * Compatible with Pilosa 0.8.x and 0.9.x.
+    * Checks the server version for Pilosa server compatibility. Passing `SkipVersionCheck()` to `NewClient` disables that.
+    * Supports string row and column IDs. This feature requires Pilosa Enterprise.  
```

Comment:
We should remove this since we're changing our enterprise strategy. We should probably just say "this feature is not yet supported in Pilosa (so don't use it yet!)"

Reply:
I've removed that line altogether to avoid confusion.